mesonbuild/dependencies/base.py

```diff
@@ -1085,6 +1085,7 @@ def __init__(self, name: str, environment: Environment, kwargs, language: T.Opti
         if self.cmakeinfo is None:
             raise self._gen_exception('Unable to obtain CMake system information')
 
+        version = kwargs.get('cmake_version', '')
```

Comment:
There should be a check that `version` is a `str` (since this is user input)

Reply:
Should be fixed in the latest version.